lib/graphql-hive/configuration.rb

```diff
@@ -35,7 +35,7 @@ def validate!
           @enabled = false
           @report_schema = false
         end
-        if @report_schema && (@reporting.dig(:author) || !@reporting.dig(:commit))
+        if @report_schema && !(@reporting.dig(:author) || @reporting.dig(:commit))
```

Comment:
Caught a bug here! This is still unreleased code, so we don't need to push a new version.

Reply:
From the logs, it seems like we want both author **and** commit to be present, so I think this should be:
```suggestion
        if @report_schema && !(@reporting.dig(:author) && @reporting.dig(:commit))
```
So if only one is present, and the other not `!(true || false) = false`, we we should turn off @report_schema